scripts/technical-debt-metrics.sh

```diff
@@ -77,7 +77,7 @@ titlesPathsAndRegexes=(
   "porting notes"                  "*"      "Porting note"
   "backwards compatibility flags"  "*"      "set_option.*backward"
   "skipAssignedInstances flags"    "*"      "set_option tactic.skipAssignedInstances"
-  "adaptation notes"               "*"      "adaptation_note"
+  "adaptation notes"               ":^Mathlib/Tactic/AdaptationNote.lean"      "^[· ]*#adaptation_note"
```

Comment:
Looks like we stop trying to align lines if they don't fit (cf. `maxHeartBeats modifications`):
```suggestion
  "adaptation notes"               ":^Mathlib/Tactic/AdaptationNote.lean" "^[· ]*#adaptation_note"
```